src/library/basetracktablemodel.cpp

```diff
@@ -109,7 +109,7 @@ int BaseTrackTableModel::s_bpmColumnPrecision =
         kBpmColumnPrecisionDefault;
 bool BaseTrackTableModel::s_keyColorsEnabled = kKeyColorsEnabledDefault;
 ColorPalette BaseTrackTableModel::s_keyColorPalette =
-        mixxx::PredefinedColorPalettes::kDefaultKeyColorPalette;
+        mixxx::predefinedcolorpalettes::get().defaultKeyColorPalette;
```

Comment:
In #13594 we have an `std::optional` which makes redundant initialization here obsolete.  
Let's mere #13594 first.